### DISCUSSION TITLE
Why we shifted to Sendgrid over Nodemailer

### DISCUSSION THREAD

**Discussion by aashutoshrathi** [iiitv/getmein-web#135] [Q&A] [answered]
Had a minor discussion regarding it with @AmanRaj1608 
He will fill the answer as explained.

**Comment by AmanRaj1608** [comment, marked as answer]
Doubt: - We were using Nodemailer SMTP (which is free up to Gmail req) then why we shifted to Sendgrid (100 emails per day)?

Some issues while we were using Nodemailer:-
- Giving email password in production env which can be accessed by other people working on the production.
- Google sending emails about it
- Rejects nodemailer logins sometimes
